### ISSUE TITLE
Windows Installer marked as virus

### ISSUE THREAD

**Issue by GustavJones** [KRTirtho/spotube#613]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When the Windows installer is downloaded onto my computer, my browser (Brave) and Windows Defender marked the file as a threat and deleted it.

### Expected Behavior

File downloaded successfully without any anti virus threat detection within the installer

### Steps to reproduce

1. Download the Windows Setup .exe file
2. Wait for browser or Windows Defender to mark as a threat and delete the file

### Operating System

Windows 11 Pro

### Spotube version

v3.0.1

### Installation source

GitHub Releases (Binary)

**Comment by KRTirtho** [comment]
There's no certificates so obviously Microsoft decides it's a virus

But If you're concerned you can use Winget or chocolatey to install

**Comment by KRTirtho** [comment]
There's no certificates so obviously Microsoft decides it's a virus

But If you're concerned you can use Winget or chocolatey to install

**Comment by hessianerd** [comment]
is it possible to provide a standalone portable app for windows? Like what they offer at https://www.liberkey.com/en.html

**Comment by KRTirtho** [comment]
Yea it's possible. But it doesn't really make sense for a music player

**Comment by hessianerd** [comment]
> Yea it's possible. But it doesn't really make sense for a music player

Its really helpful to not have to install.  It would allow me to use the music player on my work computer for example.

**Comment by iamgmujtaba** [comment]
This issue still exists. Any suggestion?

![image](https://github.com/KRTirtho/spotube/assets/33286377/915e1d30-c1e7-4d27-814a-fb8c7a1c1601)

**Comment by stefanov-stefan** [comment]
![image](https://github.com/KRTirtho/spotube/assets/1288402/fdb39c35-516d-4ec2-95b2-a54519d00013)

**Comment by B1ack3ye** [comment]
Still marked as Virus.
![image](https://github.com/KRTirtho/spotube/assets/63259739/2d2d697e-56e3-4114-a773-772ddd04e8ff)
![image](https://github.com/KRTirtho/spotube/assets/63259739/5044ced8-b89f-4196-9217-6d4d123f7bc6)

**Comment by KRTirtho** [comment]
Yeah, awesome, our totally trustworthy vendors and super nice corporate company Google and Microsoft flagged it as malicious.

**Comment by ManIWishIHadTacos** [comment]
> Yeah, awesome, our totally trustworthy vendors and super nice corporate company Google and Microsoft flagged it as malicious.

Please just get your app signed already.

**Comment by a-p-jo** [comment]
> > Yeah, awesome, our totally trustworthy vendors and super nice corporate company Google and Microsoft flagged it as malicious.
> 
> Please just get your app signed already.

It's not that simple. And signing is a lie.

**Comment by a-p-jo** [comment]
> There's no certificates so obviously Microsoft decides it's a virus
> 
> But If you're concerned you can use Winget or chocolatey to install

Note that this happens for a winget install too.

**Comment by J3r0github** [comment]
Detected as malicious by Crowdstrike Falcon... Now I don't believe it is malicious but am unable to use right now, which... sucks.

**Comment by ManIWishIHadTacos** [comment]
You can look at the source if you don't trust it
